test/asserts/accept-keys-assert_test.js

```diff
@@ -0,0 +1,93 @@
+
+/**
+ * Module dependencies.
+ */
+
+import { Assert as BaseAssert, Violation } from 'validator.js';
+import AcceptKeysAssert from '../../src/asserts/accept-keys-assert';
+import should from 'should';
+
+/**
+ * Extend `Assert` with `AcceptKeysAssert`.
+ */
+
+const Assert = BaseAssert.extend({
+  AcceptKeys: AcceptKeysAssert
+});
+
+/**
+ * Test `AcceptKeysAssert`.
+ */
+
+describe('AcceptKeysAssert', () => {
+  it('should throw an error if given input is not a plain object', () => {
+    [[], '', 123].forEach((value) => {
+      try {
+        new Assert().AcceptKeys(['foo', 'bar']).validate(value);
+
+        should.fail();
+      } catch (e) {
+        e.should.be.instanceOf(Violation);
+        e.violation.value.should.equal('must_be_a_plain_object');
+      }
+    })
+  });
+
+  it('should throw an error if given object has unaccepted keys', () => {
```

Comment:
This test is not correct. If `foo` is accepted, this test must pass. Extra keys are always allowed, otherwise just use `EqualKeys`.